src/components/Login/Login.jsx

```diff
@@ -63,16 +63,12 @@ export class Login extends React.Component {
     }
 
     render() {
+        const letters = ["T", "e", "l", "l", "T", "a", "l", "e", "s"];
         return (
             (!this.props.cookie) ?
                 <div className={styles.loginPage}>
-                     <div className={styles.foo}>
-                        <span className={styles.letter} data-letter="D">D</span>
-                        <span className={styles.letter} data-letter="i">i</span>
-                        <span className={styles.letter} data-letter="X">X</span>
-                        <span className={styles.letter} data-letter="i">i</span>
-                        <span className={styles.letter} data-letter="t">t</span>
-
+                    <div className={styles.foo}>
+                        {letters.map((letter, key) => <span key={key} className={styles.letter} data-letter={letter}>{letter}</span>)}
```

Comment:
I'm not saying you should change this (hopefully you won't need to change the name again) but I was thinking that you'd have a string rather than an array of letters. Then you'd split the string before mapping.

Reply:
oh I see, that makes sense!